wren-ai-service/src/web/v1/services/semantics_description.py

```diff
@@ -25,7 +25,7 @@ class Error(BaseModel):
             message: str
 
         id: str
-        status: Literal["generating", "finished", "failed"] = None
+        status: Literal["generating", "finished", "failed"] = "generating"
```

Comment:
I suggest we don't add default value here, instead we give the status in the beginning of the generate method